cli/unipipe/blueprints/terraform-runner-hello-world-service.ts

```diff
@@ -0,0 +1,156 @@
+      longDescription: "This simple service renders recipes in your git repository. It uses a terraform script that is executed by terraform-runner. Have a look in the terraform/ folder of your git repository to find out more."
+      providerDisplayName: "Pizza Inc."
+      documentationUrl: "https://docs.microsoft.com/en-us/azure/devops/?view=azure-devops"
```

Comment:
d: Should we skip the URLs here. I guess links to Azure DevOps are more confusing than helpful here?